libcrux-ml-kem/benches/ml-kem.rs

```diff
@@ -107,7 +107,8 @@ pub fn pk_validation(c: &mut Criterion) {
     init!(mlkem1024, "PK Validation", c);
 }
 
-pub fn encapsulation(c: &mut Criterion) {
+#[target_feature(enable = "avx2")]
```

Comment:
These changes need to get undone in this file

Reply:
Done